src/main/resources/application.properties

```diff
@@ -65,11 +65,11 @@ groupings.api.failure=FAILURE
 groupings.api.current_user=current_user
 
 # Attributes names returned by Grouper in WsSubject.
-groupings.api.person_attributes.username=uid
-groupings.api.person_attributes.uhuuid=uhUuid
-groupings.api.person_attributes.first_name=givenName
-groupings.api.person_attributes.last_name=sn
-groupings.api.person_attributes.composite_name=cn
+groupings.api.subject_attributes.uid=uid
+groupings.api.subject_attributes.uhuuid=uhUuid
+groupings.api.subject_attributes.first_name=givenName
+groupings.api.subject_attributes.last_name=sn
+groupings.api.subject_attributes.composite_name=cn
```

Comment:
I think these are not being used and can be removed. However, if you think there would be a good place to use these properties please use them.